docs/zh/development/apollo-release-guide.md

```diff
@@ -0,0 +1,103 @@
+# 1. 升级版本
+在 Apollo 工程下全文替换版本号，例如 1.9.1-SNAPSHOT 升级到 1.9.1。替换过程中，一定要注意非版本号的地方替换。
+# 2. 撰写发布报告
+两个版本之间每一个 PR 都会记录在 [https://github.com/ctripcorp/apollo/blob/master/CHANGES.md](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md) 里，所以撰写发布报告只需要从 CHANGES.md 里提取即可。发布报告参考：[https://github.com/ctripcorp/apollo/releases/tag/v1.8.0](https://github.com/ctripcorp/apollo/releases/tag/v1.8.0)
+​
+
+# 3. 版本验证
+版本验证主要包含三方面的验证：
+
+1. 新引入的代码变更验证，例如新功能、bugfix
+1. Apollo 核心主流程验证，包括：配置发布，动态推送，灰度推送等
+1. 升级过程验证，包括：经典部署模式、docker 模式、k8s 模式
+
+
+
+# 4. 版本发布
+## 4.1 打 tag
+
+1. 拉取 master 最新代码
+   1. git pull origin master  
+2. 打 tag
+   1. git tag v1.9.0
+3. push tag
+   1. git push origin tag v1.9.0
+## 4.2 打包
+### 4.2.1 前置检查
+在打包之前检查本地环境, mvn -v 确保 java 版本是 1.8, 例如以下输出：
+> mvn -v
+> Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)Maven home: /usr/local/Cellar/maven/3.8.1/libexec
+> Java version: 1.8.0_301, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_301.jdk/Contents/Home/jre
+> Default locale: zh_CN, platform encoding: UTF-8
+> OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"
+
+### 4.2.2 打包
+在 ${apollo_home}/scripts/ 目录下执行：
+> ./build.sh
+
+如果报以下错误：
+> zsh: ./build.sh: bad interpreter: /bin/sh^M: no such file or directory
+
+则需要执行以下命令转换成 unix
+> brew install dos2unix
+> dos2unix build.sh
+
+### 4.2.3 计算构建包的 checksum
+在 scripts 目录下执行以下三个命令
+​
+
+计算 configservice checksum
+> shasum ../apollo-configservice/target/apollo-configservice-1.9.0-github.zip > ../apollo-configservice/target/apollo-configservice-1.9.0-github.zip.sha1
+
+计算 adminservice checksum
+> shasum ../apollo-adminservice/target/apollo-adminservice-1.9.0-github.zip > ../apollo-adminservice/target/apollo-adminservice-1.9.0-github.zip.sha1
+
+计算 portal checksum
+> shasum ../apollo-portal/target/apollo-portal-1.9.0-github.zip > ../apollo-portal/target/apollo-portal-1.9.0-github.zip.sha1
+
+## 4.3 创建 pre-release
+github 创建 pre-release
+![image.png](https://cdn.nlark.com/yuque/0/2021/png/94396/1629015924779-f517e179-2711-4ccb-ab46-1fb69ca2f191.png#clientId=u9690858a-d2d3-4&from=paste&height=641&id=u4161f0d9&margin=%5Bobject%20Object%5D&name=image.png&originHeight=1282&originWidth=3564&originalType=binary&ratio=1&size=258348&status=done&style=none&taskId=ue15b5715-83f8-4ff2-9c6a-285d1c11989&width=1782)
```

Comment:
I found sometimes the image won't display, is it better to upload the image to github?

Reply:
done